openai/ai-screenwriter-template/web/routes/signed-in.tsx

```diff
@@ -6,6 +6,8 @@ import {
 } from "@gadgetinc/react";
 import { api } from "../api";
 import { useState } from "react";
+import { ChatRequestBody } from "../../api/routes/POST-chat";
```

Comment:
I don't think we should be sharing code between backend and frontend, even if it is just types